odoo/addons/base/tests/test_module.py

```diff
@@ -94,3 +94,104 @@ def test_missing_license(self):
             manifest = load_manifest(self.module_name)
         self.assertEqual(manifest['license'], 'LGPL-3')
         self.assertIn("Missing `license` key", capture.output[0])
+
+    def test_latent_dependency_issue_data_loaded(self):
```

Comment:
For technical reasons, you may not test module operations (install, upgrade, uninstall) in unit tests.  Such tests must be done as standalone tests.  See odoo/tests/test_module_operations.py for instance.